tools/executors/node/schema.json

```diff
@@ -161,6 +161,14 @@
       "type": "string",
       "description": "Name of the main output file. (defaults to `main.js`)",
       "default": "main.js"
+    },
+    "preBuildCommands": {
```

Comment:
Think we should use run-commands instead 👍

Reply:
Agreed